src/udp_wrap.cc

```diff
@@ -219,6 +221,38 @@ void UDPWrap::Bind6(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void UDPWrap::BufferSize(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  UDPWrap* wrap;
+  ASSIGN_OR_RETURN_UNWRAP(&wrap,
+                          args.Holder(),
+                          args.GetReturnValue().Set(UV_EBADF));
+
+  CHECK(args[0]->IsUint32());
+  CHECK(args[1]->IsUint32());
+  int size = (int)args[0].As<Uint32>()->Value();
```

Comment:
I think we try to avoid C style casts in C++ code. It also seems a bit odd that we would require a Uint32 and then cast it to an int.